NetSDK/Services/Client/Classes/SplitFactory.cs

```diff
@@ -43,7 +43,19 @@ private void BuildSplitClient()
             }
             else
             {
-                client = new SelfRefreshingClient(apiKey, options);
+                if (options.RedisConfig != null)
```

Comment:
Include MODE config (3 possible values: Standalone (default, in memory), Consumer (redis), Producer (no action))
And Cache Adapter
See: http://docs.split.io/docs/ruby-sdk-overview#section-consumer-mode